content/bn/docs/reference/glossary/node.md

```diff
@@ -0,0 +1,18 @@
+---
+title: নোড
+id: node
+date: 2018-04-12
+full_link: /docs/concepts/architecture/nodes/
+short_description: >
+  নোড হলো কুবারনেটিসে একটি ওয়ার্কার মেশিন।
+
+aka:
+tags:
+- fundamental
+---
+নোড হলো কুবারনেটিসে একটি ওয়ার্কার মেশিন।
+<!--more-->
+
+একটি ওয়ার্কার নোড একটি ভার্চুয়াল মেশিন বা ফিজিক্যাল মেশিন হতে পারে, একটি ক্লাস্টারের উপর নির্ভর করে । এটির {{< glossary_tooltip text="Pods" term_id="pod" >}} সহজে চালানোর জন্য পরিযোজনীয় সকল লোকাল ডেমন(daemons) বা সেবা আছে এবং এটি কন্ট্রোল প্লেন দ্বারা পরিচালিত হয়। একটী নোডের ডেমনগুলোতে থাকে {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}, {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}, এবং একটি কন্টেনার রানটাইম যা {{< glossary_tooltip text="CRI" term_id="cri" >}} সম্পাদন করে যেমন {{< glossary_tooltip term_id="docker" >}}
+
+প্রাথমিক কুবারনেটিস সংস্করণে, নোডগুলি "মিনিয়নস"(Minions) হিসেবে পরিচিত ছিল।
```

Comment:
প্রাথমিক কুবারনেটিস সংস্করণে, নোডগুলি "Minions" হিসেবে পরিচিত ছিল।

Reply:
I replaced the text